### DISCUSSION TITLE
Set default yaml front matter

### DISCUSSION THREAD

**Discussion by rosikand** [quarto-dev/quarto-cli#2727] [Q&A] [answered]
Hello, 

Say I want to have the same base YAML front matter for all posts in `/posts` which is linked to a listing page. 

For example, something like this: 

```
---
format: 
  html: default
  pdf: default
  docx: default
---
```

How can I set this to be the default without having to explicitly type it out for all `.md`/`.qmd` files in `/posts`?

**Comment by cderv** [comment, marked as answer]
You should have a look at project basics about shared metadata :https://quarto.org/docs/projects/quarto-projects.html#shared-metadata

You can have shared metadata for all qmd file under a subdirectory
